### PR TITLE
Added support for python 3

### PR DESCRIPTION
**Pull request by DSilence**

### CODE REVIEW

File: monitorrent/plugin_managers.py [hunk 1]
```diff
@@ -138,14 +140,14 @@ def get_watching_topics(self):
         return watching_topics
 
     def execute(self, engine):
-        for name, tracker in self.trackers.iteritems():
+        for name, tracker in self.trackers.items():
```
Comment (werwolfby) [inline monitorrent/plugin_managers.py]: Should we use: 
```
list(self.trackers.items())
```
as well or we can remove list everywhere?

Comment (DSilence) [inline monitorrent/plugin_managers.py]: That's how it works:
In Python 2, the methods items(), keys() and values() used to "take a snapshot" of the dictionary contents and return it as a list. It meant that if the dictionary changed while you were iterating over the list, the contents in the list would not change.

In Python 3, these methods return a view object whose contents change dynamically as the dictionary changes. Therefore, in order for the behavior of iterations over the result of these methods to remain consistent with previous versions, an additional call to list() has to be performed in Python 3 to "take a snapshot" of the view object contents.

TL; DR: We can remove list everywhere, assuming this will not break the functionality, however it's safer with list(), cause it keeps the behavior the same.

Comment (werwolfby) [inline monitorrent/plugin_managers.py]: But looks like in this particular line you forgot about list

**Comment by werwolfby** [comment]
And **lxml-3.6.0-cp35-cp35m-win32.whl** shouldn't be commited to repository.

File: monitorrent/plugin_managers.py (hunk 1 above)
Comment (DSilence) [inline monitorrent/plugin_managers.py]: Done by autoconversion

**Comment by DSilence** [comment]
1) Can't be downloaded by wget.
2) No python 3.5 version for windows on pipy.

**Comment by werwolfby** [comment]
Can be downloaded from here for now: http://www.edna-site.org/pub/wheelhouse/lxml-3.6.0-cp35-cp35m-win32.whl

**Comment by DSilence** [comment]
Thanks for the tip, I'll use that.

File: monitorrent/plugin_managers.py (hunk 1 above)
Comment (werwolfby) [inline monitorrent/plugin_managers.py]: Line 89 has list() of items, while line 143 doesn't.
May be a bug in conversion tool?

Comment (DSilence) [inline monitorrent/plugin_managers.py]: Could be. I'll add list here.

**Comment by werwolfby** [comment]
And please rebase it to the current develop.

**Comment by DSilence** [comment]
Done
